apis/Google.Cloud.Logging.Console/SampleApp/Pages/Error.cshtml.cs

```diff
@@ -0,0 +1,27 @@
+using System.Diagnostics;
```

Comment:
No licensences on these files?
Are most of these just from the project template?